scripts/build-timeseries-index.ts

```diff
@@ -0,0 +1,363 @@
+// scripts/build-timeseries-index.ts
```

Comment:
NB: In a future PR, we might consider adding CI that checks that this runs cleanly

Reply:
This is in `prebuild` and `predev`, so it runs as part of the build process